app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime

```

Comment:
It's not correct imports order first
![image](https://user-images.githubusercontent.com/104218550/198878180-b8810517-1c57-4636-ac04-033b7869e068.png)